tests/image/image_test.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import (division, absolute_import, unicode_literals,
+                        print_function)
+
+import unittest
+
+from file_metadata.image.image import ImageFile
+from tests.utilities import fetch_file, mock
```

Comment:
This file contains unused source code.

*PyUnusedCodeBear, severity NORMAL, section `python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tests/image/image_test.py
+++ b/tests/image/image_test.py
@@ -6,7 +6,8 @@
 import unittest
 
 from file_metadata.image.image import ImageFile
-from tests.utilities import fetch_file, mock
+from tests.utilities import fetch_file
+from tests.utilities import mock
 
 
 class ImageFileTest(unittest.TestCase):
```